tests/crypto/read_file.c

```diff
@@ -228,12 +228,8 @@ oe_result_t read_mod(char* filename, uint8_t* mod, size_t* mod_size)
 
 oe_result_t read_mixed_chain(char* chain, char* chain1, char* chain2)
 {
-    char chain_temp[max_cert_chain_size * 2];
-    OE_UNUSED(chain);
-
-    strcat(chain_temp, chain1);
-    strcat(chain_temp, chain2);
-    chain = chain_temp;
+    strcat(chain, chain1);
+    strcat(chain, chain2);
```

Comment:
Consider using **strncat()**?

Reply:
Good suggestion, switched to using `oe_strlcat()` as a better semantic fit for the crypto tests more broadly.

Reply:
Thanks!